### PR TITLE
fix: Add deprecation of Windows 32-bit and Python 2.7 support in README file

### LINKED ISSUE #2022: Announce deprecation of Windows 32-bit and Python 2.7 support
#1435 # Description

We need to announce that SCTK v3.2 will be the last to support Windows 32-bit or Python 2.7. 
We probably need to post this prominently in at least two places:
- README on GitHub
-  A new Announcement or similar section in RTD.
The announcement should be clear about whether there will be any exceptions for critical bugs.

### PR DESCRIPTION
**Pull request by afzal442**
Fixes nexB#2022

This PR adds WARNING note into the README file about WSL requirement

Signed-off-by: Afzal Ansari afzal442@gmail.com

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -113,6 +113,7 @@ Before installing ScanCode make sure that you have installed the prerequisites
 properly. This means installing Python (Python 3.6+ is required).
 See `prerequisites <https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#prerequisites>`_
 for detailed information on the support platforms and Python versions.
+Note: Native Windows will be deprecated in the future in favor of Windows Subsystem for Linux. This means that WSL https://docs.microsoft.com/en-us/windows/wsl/install-win10 would become a requirement on Windows platform.
```
Comment (pombredanne) [inline README.rst]: Here is suggested improved way with proper wrapping.
```suggestion
Note: Native Windows will be deprecated in the future in favor of Windows
Subsystem for Linux (WSL): WSL and Windows 10 will become a
requirement on Windows.
```

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thanks! Do you mind to add your DCO signoff in the commit message?

File: README.rst (hunk 1 above)
Comment (afzal442) [reply README.rst]: Hi @pombredanne. Thanks for your review. I wanna just commit this.

**Comment by afzal442** [comment]
@pombredanne I am just surprised to see that my DCO test case is failing everytime after commiting. Plz assist with that. Thanks.

**Comment by afzal442** [comment]
@pombredanne How is it failing over and over again? Could you help me with this? Thanks.

**Comment by AyanSinhaMahapatra** [comment]
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git 

Squash all your commits in one and add the signoff there. Your first commit `Updated the README file` doesn't have a signoff that's why it's failing. Also, see https://aboutcode.readthedocs.io/en/latest/contributing/writing_good_commit_messages.html

We should also probably mention `WSL 2` explicitly.

**Comment by pombredanne** [comment]
@afzal442 I am closing this for now, since this is missing a DCO signoff. Please feel free to reopen. Thanks!

**Pull request closed without merging**